src/gwc/src/main/java/org/geoserver/gwc/layer/GeoServerTileLayer.java

```diff
@@ -370,7 +370,7 @@ public PublishedInfo getPublishedInfo() {
         return publishedInfo;
     }
 
-    private ResourceInfo getResourceInfo() {
+    public ResourceInfo getResourceInfo() {
```

Comment:
This might be a problem if we ever want to contribute this to community, as it is an API change; It would be be better to just replicate the functionality where it is needed.